Before: 412233323e7060dfdfc1775176cfa9e7f781566c2dcd5ce13f5c3dab063630ec
After: 9b49c54dd5e08528fa5b3f21c95fb89aa3e82057afa908e8853fd10704f95181
Patch: 7d79f486c13f8a440b418f05e9fc4579e1925ab6cad050a90bf24c69332cda88
--- %name%.txt.tt	2012-09-11 13:27:33.401301034 +1000
+++ %name%.txt.tt	2012-09-11 13:27:33.354634088 +1000
@@ -1,4 +1,3 @@
 My name is <%= name %>.
 I'm <%= age %> years old.
 Project name <%= project_name %>
-Bye.
--- corvid-features/template2.rb	2012-09-11 13:27:33.401301034 +1000
+++ corvid-features/template2.rb	2012-09-11 13:27:33.354634088 +1000
@@ -4,4 +4,4 @@
 }
 
 def name; 'George' end
-def age; 23 end
+def age; 22 end
